### ISSUE TITLE
Enhancement request: Add support for Pypykatz

### ISSUE THREAD

**Issue by voidnecron** [xFreed0m/KatzKatz#4]
Hi,

As mentioned in the other issue, request for supporting Pypykatz log formats.

Example:

        == MSV ==
                Username: administrator
                Domain: testdomain
                LM: NA
                NT: ebacxxxxxxxxxxxxxxxxxxxxxxxxf7fb
                SHA1: ba68xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx5c7d
        == MSV ==
                Username: NA
                Domain: NA
                LM: NA
                NT: ebacxxxxxxxxxxxxxxxxxxxxxxxxf7fb
                SHA1: ba68xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx5c7d
        == WDIGEST [2055ef79]==
                username administrator
                domainname testdomain
                password None
        == SSP [2055ef79]==
                username 
                domainname 
                password None
        == Kerberos ==
                Username: administrator
                Domain: testdomain.local
                Password: None
        == WDIGEST [2055ef79]==
                username administrator
                domainname testdomain
                password None

Regex should be something like these:
```
re.compile(r"\s*\s+Username:\s+((?!\(null\)).+)\s*(?!\$)", re.M)
re.compile(r"\s*\s+Password:\s+((?!\(null\)).+)\s*(?!\$)", re.M)
re.compile(r"\s*\s+Domain:\s+((?!\(null\)).+)\s*(?!\$)", re.M)
re.compile(r"\s*\s+NT:\s+((?!\(null\)).+)\s*(?!\$)", re.M)
```

**Comment by xFreed0m** [comment]
Hey @voidnecron ,
I've had some time to work on that, and i think i got it done!
I have limited testing options in my lab, so please, test the new branch:
https://github.com/xFreed0m/KatzKatz/blob/pypykatz_support/katzkatz.py

waiting to hear back if everything is working as expected.

**Comment by xFreed0m** [comment]
implemented in #5
